ADVANCED.md

```diff
@@ -314,21 +314,38 @@ customizations, like simply hiding the search button, this is often overkill.
 ```
 
 You can also just customize the input section of the search box. Useful for things
-like hiding the submit button:
+like hiding the submit button or rearranging dom structure:
```

Comment:
Super minor grammar nit: DOM should probably be capitalized here / for consistency (it's capitalized elsewhere on the page)

Reply:
Good point! Sorry, hit the merge button.